e2e/test/scenarios/onboarding/urls.cy.spec.js

```diff
@@ -127,7 +127,7 @@ describe("URLs", () => {
         `${getFullName(admin)}'s Personal Collection`,
       );
 
-      cy.visit(`/collection/9-${getUsersPersonalCollectionSlug(normal)}`);
+      cy.visit(`/collection/7-${getUsersPersonalCollectionSlug(normal)}`);
```

Comment:
These collection IDs were changed to match new personal collection IDs. Personal Collections appear to be created (if they don't already exist) when you fetch a users data and the personal_collection_id is hydrated. Since we changed the order the users return from the database (the change was a sql change), then we're also changing the order that the personal collections are created. Some additional information can be found [here](https://github.com/metabase/metabase/blob/52bf5d5a0ae4f497d675e0519c89c5cb0ae3ddae/e2e/snapshot-creators/default.cy.snap.js#L127)